src/validation.rs

```diff
@@ -92,7 +92,7 @@ pub enum HedgeCorrectness {
     EdgeIsAbsent,
     /// The radial chain has more than 100 elements. This is not, strictly speaking, an error
     /// but it's extremely suspicious.
-    LongRadialChain,
+    LongRadialChain(HedgeId),
```

Comment:
This time, this change does make it a breaking change

Reply:
There's no problems with it. Just a reminder if you're going to publish you'd need to bump the major version number - unless of course you specifically exclude this `validation` module from the crate's semver guarantees

Reply:
> There's no problems with it. Just a reminder if you're going to publish you'd need to bump the major version number - unless of course you specifically exclude this `validation` module from the crate's semver guarantees

I am using this convention since the project is very young still

> This guide uses the terms “major” and “minor” assuming this relates to a “1.0.0” release or later. Initial development releases starting with “0.y.z” can treat changes in “y” as a major release, and “z” as a minor release. “0.0.z” releases are always major changes. This is because Cargo uses the convention that only changes in the left-most non-zero component are considered incompatible.

https://doc.rust-lang.org/cargo/reference/semver.html